cloudfour/cloudfour.com-patterns
# Improve rhythm object and its docs

## Overview

I thought #1866 would un-block a WordPress task I was working on, but I realized once I started actually implementing it that what I _really_ needed was a `--lavish-headings` modifier. While attempting to add that, I got a bit tangled up in this component's existing documentation, so I decided to simplify the examples down to two demos. Along the way I also added `amount` and `heading_amount` properties so we don't have to use string interpolation to format class names.

## Screenshots

![Screenshot 2022-06-22 at 10-22-52 objects-rhythm--example](https://user-images.githubusercontent.com/69633/175099470-3d2859a1-1740-4e76-bffe-95ce3c07590c.png)

## Testing

On [the deploy preview](https://deploy-preview-1868--cloudfour-patterns.netlify.app/)…

1. Proof-read [the revised rhythm docs](https://deploy-preview-1868--cloudfour-patterns.netlify.app/?path=/docs/objects-rhythm--example).
1. View [the default example story](https://deploy-preview-1868--cloudfour-patterns.netlify.app/?path=/story/objects-rhythm--example), mess with the controls and observe that the outcome works as described.
1. View [the nesting story](https://deploy-preview-1868--cloudfour-patterns.netlify.app/?path=/story/objects-rhythm--nesting), mess with the controls and observe that the outcome works as described.
